7-layouts/layout.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Random topic web page">
+        <meta name="keywords" content="web developer, web desing, learning, project">
+        <meta name="author" content="Jaime Moreno"> 
+        <link rel="stylesheet" href="styles.css">
+        <title>Layouts</title>
+    </head>
+
+    <body>
+      <header>
+          <p class="top-1">Grid into the future</p>
+          <h1 class="head-1">
+              Hello
+          </h1>
+          <p class="lorem">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make
+            a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting,
+            remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing
+            Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions
+            of Lorem Ipsum.</p>
+      </header>  
+      <article>
+            <p class="art-1">Title of article</p>
+            <p class="art-2">Title of article</p>
+            <p class="art-3">Title of article</p>
+            <p class="art-4">Title of article</p>
+            <p class="art-5">Title of article</p>
+            <p class="art-6">Title of article</p>
+      </article>
```

Comment:
The layout needs content-images, I mean that you need you to use the `<img>` tag in the HTML instead of setting decoration-images (The way that you are using with css).  With the content-images  Google and screen readers will understand that you are using the tags for that reason. With css properties you are meaning that are only for decoration.
What if you try something like this:
```
     <main>
          <article>
              <h2>Dog 1</h2>
              <img src="your-url" alt="image-description">
          </article>
          <!--Same with the others...-->
      </main>
```